tox.ini

```diff
@@ -1,14 +1,17 @@
 [tox]
 isolated_build = true
-envlist = isort, black, flake8, pylint, py27, py36, py37, py38
+envlist = isort, black, flake8, pylint, py37, py38, py39, py310
 
 
 [default]
 setenv =
     PACKAGE=webpub_manifest_parser
+    PYTHONPATH=./src/
 allowlist_externals =
     python
     make
+    poetry
+    pytest
```

Comment:
Is pytest needed here? Its installed in the virtualenv, so I don't think it needs to be in the externals list